internal/broker/broker.go

```diff
@@ -446,6 +446,7 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, session *sessionInfo
 			slog.Error(err.Error())
 			return AuthRetry, errorMessage{Message: "could not authenticate user remotely"}
 		}
+		slog.Debug("Got token with scopes: " + t.Extra("scope").(string))
```

Comment:
If the field is nil or can’t be casted to a string from what we received from the API, this will panic. We should check the cast was successful first, printing some logs if it wasn’t with the "%v" of t.Extra("scope") if this is not the case.

Reply:
Good catch, fixed